README.md

```diff
@@ -394,6 +394,53 @@ treeDecoder.decode({
 // Output: Err({error: "<Node<string>> decoder failed at key 'children' with error: <Node<string>[] | isUndefined> decoder failed because null can't be decoded with any of the provided oneOf decoders"})
 ```
 
+### JsonDecoder.optional
+
+> `optional<a>(decoder: Decoder<a>): Decoder<a | undefined>`
+
+Creates a decoder that returns a default value on failure.
```

Comment:
You say here `returns a default value` but the decoder doesn't have a default valuer param like, for instance, the `failover` decoder. Do we want to add the default value param?